src/components/store.tsx

```diff
@@ -198,13 +198,13 @@ export const StoreProvider: FC<StoreProviderProps> = ({ children, dbName, docId
     if (db) {
       db.upsert(docId, function upsertFn(dbDoc: any) {
         let result = {...dbDoc, ...newDoc};
-        if (!result.meta_) {
-          result.meta_ = {
+        if (!result.metadata_) { // DB datastructure change
```

Comment:
If the dev team deletes our own data, we should not need this test in the code since it is guaranteed by  the `putIfNotExists` call above.